exercises/practice/rational-numbers/rational-numbers.test.ts

```diff
@@ -150,17 +180,17 @@ describe('Exponentiation of a rational number', () => {
 describe('Exponentiation of a real number to a rational number', () => {
   xit('Raise a real number to a positive rational number', () => {
     const expected = 16.0
-    expect(new Rational(4, 3).expreal(8)).toEqual(expected)
+    expect(new Rational(4, 3).expreal(8)).toBeCloseTo(expected, 10)
   })
 
   xit('Raise a real number to a negative rational number', () => {
     const expected = 1.0 / 3.0
-    expect(new Rational(-1, 2).expreal(9)).toBeCloseTo(expected, 15)
+    expect(new Rational(-1, 2).expreal(9)).toBeCloseTo(expected, 10)
```

Comment:
Turned the number of relevant digits down because on my machine this fails:
```js
expect(8**(4/3)).toBeCloseTo((8**(1/3))**4, 15)
```

Reply:
```text
15.999999999999998
16.000000000000000
```

Hehe.